app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
i detected that hardcoded list of modes does not affect to precompile.
at the 4.x.x version was added forced build-in assets.precompile...
```ruby
module Ace
  module Rails
    class Engine < ::Rails::Engine
      initializer 'ace-rails-ap.assets.precompile' do |app|
        app.config.assets.precompile += %w[ace/worker-*.js ace/mode-*.js]
      end
    end
  end
end
```
looks like i should revert to previous version of gem.